server/subsonic/stream.go

```diff
@@ -90,6 +96,27 @@ func (api *Router) Download(w http.ResponseWriter, r *http.Request) (*responses.
 		return nil, err
 	}
 
+	maxBitRate := utils.ParamInt(r, "bitrate", 0)
+	format := utils.ParamString(r, "format")
+
+	if format == "" {
+		if conf.Server.DefaultDownloadTranscoding {
+			// if we are not provided a format, see if we have requested transcoding for this client
```

Comment:
Sorry, but I didn't get the purpose of this config when I first reviewer your PR. I think the previous name (`AutoTranscodeDownload `) actually makes more sense.